### PR TITLE
collected new content

### PR DESCRIPTION
**Pull request by alexandrtovmach**
Regarding discussion #72 this PR contain the newest docs content

@zeke Could you take a look

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
   "private": true,
   "version": "1.0.0",
   "nodeVersions": {
-    "v10.x": "v10.5.0",
-    "v8.x": "v8.11.3",
-    "v6.x": "v6.14.2"
+    "v12.x": "v12.6.0",
```
Comment (TiagoDanin) [inline package.json]: Missing update the test (`defines nodeVersions`) with the version `v12.x`.

Comment (alexandrtovmach) [reply package.json]: Sorry, could you explain more what do you mean?

Comment (alexandrtovmach) [reply package.json]: Okay, I see you mean `test.js`

**Comment by alexandrtovmach** [comment]
@TiagoDanin

**Comment by alexandrtovmach** [comment]
@zeke Done

**Pull request merged**